doc/nrf/libraries/modem/lte_lc.rst

```diff
@@ -147,6 +147,18 @@ Tracking Area Update (TAU) Pre-warning:
   * :c:enumerator:`LTE_LC_EVT_TAU_PRE_WARNING` events
   * :kconfig:option:`CONFIG_LTE_LC_TAU_PRE_WARNING_NOTIFICATIONS`
 
+DNS fallback:
+  The :kconfig:option:`CONFIG_LTE_LC_DNS_FALLBACK_MODULE` Kconfig option controls the use of a fallback DNS server address.
+
+  The device may or may not receive a DNS server address by the network during PDN connection.
+  Even within the same network, the PDN connection establishment method (PCO vs ePCO) may change when the device operates in NB-IoT or LTE Cat-M1, and result in missing DNS server addresses when one method is used, but not the other.
+  Setting a fallback DNS address ensures that the device always has a DNS server address to fallback to regardless of whether the network has provided one.
+
+  This option is enabled by default.
+  If the application has configured a DNS server address in Zephyr's native networking stack, using the :kconfig:option:`CONFIG_DNS_SERVER1` Kconfig option,
+  the same server is set as the fallback address for DNS queries offloaded to the nRF9 series modem. Otherwise, the :kconfig:option:`CONFIG_LTE_LC_DNS_FALLBACK_ADDRESS` Kconfig option
+  controls the fallback DNS server address, which by default is set to Cloudflare's DNS server: 1.1.1.1.
```

Comment:
```suggestion
  If the application has configured a DNS server address in Zephyr's native networking stack using the :kconfig:option:`CONFIG_DNS_SERVER1` Kconfig option, the same server is set as the fallback address for DNS queries offloaded to the nRF9 Series modem.
  Otherwise, the :kconfig:option:`CONFIG_LTE_LC_DNS_FALLBACK_ADDRESS` Kconfig option controls the fallback DNS server address that is set to Cloudflare's DNS server: 1.1.1.1 by default.
```
Should we have nRF91 Series instead of just nRF9?

Reply:
I don't have a strong opinion here.

Reply:
`sdk-nrf` currently mentions "nRF91 Series" 422 times in docs and comments. "nRF9 series" is not mentioned at all. So I'd use what we currently have.